docs/properties-type-object-no-definition.md

```diff
@@ -0,0 +1,45 @@
+# PropertiesTypeObjectNoDefinition
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Policy-V1-03
+
+## Output Message
+
+Properties with type:object should have definition of a reference model
+
+## Description
+
+If Properties with type:object dont have a reference model defined, then the allowed types can only be primitive data types instead of type:object
```

Comment:
>If Properties with type:object dont have a reference model defined [](http://example.com/codeflow?start=0&length=66)

You can simply say - "Properties with type:object that dont reference a model definition are not allowed. ARM doesnt allow generic type definitions as this leads to bad customer experience."